scripts/waitonserver.sh

```diff
@@ -2,7 +2,7 @@
 
 echo 'waiting on healthcheck' >&2
 
-for _ in {1..6}; do
+for _ in {1..100}; do
```

Comment:
Maybe sleep for some time?

Reply:
there is `sleep 5` below already, sleeping for longer means there's gonna be delay between start and when it's detected

Reply:
Oh did't see that